docs/proposals/proposals_process_proposal.MD

```diff
@@ -0,0 +1,107 @@
+- Author notifies ansible-devel mailing list for transparency, providing link to PR.
+- Author includes commentary indicating that comments should *not* be in response to this email, but rather, community members should add comments or feedback to the pull request.
+
+2: KEEP THE PROPOSAL MOVING TOWARDS A DECISION.
+- Proposals use public meetings as a mechanism to keep them moving.
+- All proposals are decided on in a public meeting by a combination of folks with commit access to Ansible and any interested parties / users, as well as the author of the proposal. Time for approvals will be a portion of the overall schedule; proposals will be reviewed in the order received and may occasionally be deferred to the next meeting. If we are overwhelmed, a separate meeting may be scheduled.
+
+(Note: ample feedback in the comments of the proposal PR should allow for folks to come to broad consensus in one way or another in the meeting rather rapidly, generally without an actual counted vote. However, the decision should be made *in the meeting*, so as to avoid any questions around whether or not the approval of one Ansible maintain / committer reflects the opinions or decision of everyone.)
+
+- *New* proposals are explicitly added to the public IRC meeting agenda for each week by the meeting organizer for for acknowledgement of ongoing discussion and existence, and/or easy approval/rejection. (Either via a separate issue somewhere tracking any meeting items, or by adding a “meeting” label to the PR.)
+- Existing new, not-yet-approved proposals are reviewed weekly by meeting organizer to check for slow-moving/stalled proposals, or for flags from the proposal owner indicating that they'd like to have it addressed in the weeks meeting
+
+3: PROPOSAL APPROVED
+- Amendments needed to the proposal after IRC discussion should be made immediately.
+- The proposal status should be changed to Approved / In Progress in the document.
+- The proposal should be moved from /docs/proposals to a docs/roadmap folder (or similar).
+- The proposal PR comments should be updated with a note by the meeting organizer that the proposal has been accepted, and further commentary should be in the PRs implementing the code itself.
+- Proposals can also be PENDING (waiting on something), or DECLINED. 
+
+4: CODE IN PROGRESS
+- Approved proposals should be periodically checked for progress, especially if tied to a release and/or is noted as release blocking.
+- PRs implementing the proposal are recommended to link to the original proposal PR or document for context.
+5: CODE COMPLETE
+- Proposal document, which should be in docs/roadmap, should have their status updated to COMPLETE.
+- The release notes file for the targeted release should be updated with a small note regarding the feature or enhancement; completed proposals for community processes should have a follow-up mail sent to the mailing list providing information and links to the new process.
```

Comment:
In the case of the "roadmap directory", should that exist: Possibly. I think ideally we might have a directory (it could be called specs, or roadmap, or ???) for each release, or perhaps one called "current" and then past release numbers. 

Proposals that are passed on, abandoned, put on hold / deferred to the future,  etc. could each have their own directory, yes. 

One thing to consider is whether or not we want to clutter up the tree with all of this for all releases forever, or archive things to some other historical location.